lib/mzgl/gl/Graphics.h

```diff
@@ -220,12 +225,10 @@ class Graphics {
 	glm::mat4 viewProjectionMatrix;
 	friend class ScopedAlphaBlend;
 	friend struct ScopedTranslate;
-
+	friend class GraphicsAPI;
+	friend class OpenGLAPI;
```

Comment:
using `friend class` always makes me assume there is a hack somewhere - Are they really needed, or could we lean harder on making GraphicsApi a truly pure virtual interface?

Reply:
got rid of it now cheers, good point, didn't like it either